### PR TITLE
nixos/tests/k3s: fix multi-node test

### PR DESCRIPTION
**Pull request by euank**
###### Description of changes

This corrects the multi-node test after a couple recent changes which resulted in it being broken.

The `lib.toString` change was an incorrect result from a tree-wide refactor, and the aarch64 change also introduced an error in python indentation/formatting I believe.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests)

    ```
    $ nix build '.#nixosTests.k3s.multi-node'
    # passes
    $ nix build '.#nixosTests.k3s.single-node'
    # also passes
    ```

- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by superherointj — APPROVED** [review]

File: nixos/tests/k3s/multi-node.nix [hunk 1]
```diff
@@ -138,12 +138,15 @@ import ../make-test-python.nix ({ pkgs, lib, ... }:
...
 
+      is_aarch64 = "${toString pkgs.stdenv.isAarch64}" == "1"
+
       # wait for the agent to show up
       server.wait_until_succeeds("k3s kubectl get node agent")
 
       for m in machines:
-      '' # Fix-Me: Tests fail for 'aarch64-linux' as: "CONFIG_CGROUP_FREEZER: missing (fail)"
-      + lib.optionalString (!pkgs.stdenv.isAarch64) ''m.succeed("k3s check-config")'' + ''
+          # Fix-Me: Tests fail for 'aarch64-linux' as: "CONFIG_CGROUP_FREEZER: missing (fail)"
```
Comment (maxeaubrey) [inline nixos/tests/k3s/multi-node.nix:147]: I'm wondering if there's any reason we don't enable this in the kernel config on aarch64?

**Review by amaxine — APPROVED** [review]
Tests pass.

File: nixos/tests/k3s/multi-node.nix (hunk 1 above)
Comment (superherointj) [reply nixos/tests/k3s/multi-node.nix:147]: Seems the right solution. Someone has to make it work. And provide a patch for that.

Comment (maxeaubrey) [reply nixos/tests/k3s/multi-node.nix:147]: Unfortunately I'm not super familiar with the state of Linux on arm, so I can't take this on. It does seem like rebuilding my arm box with the flag flipped works "fine" and k3s appears happy, but that's not a call I'm willing to make for everyone else :sweat_smile:

Comment (superherointj) [reply nixos/tests/k3s/multi-node.nix:147]: Will check. Patch was in previous version.

**Pull request merged**